app/src/main/res/layout/content_item.xml

```diff
@@ -68,6 +68,7 @@
       android:text="@{htmlContent}"
       android:textColor="@color/oppiaPrimaryText"
       android:textSize="16sp"
+      android:textDirection="anyRtl"
```

Comment:
@veena14cs as per our initial conversation we should not use any attributes like `textAlignment`, `textDirection`, etc. I think for this PR it needs to be solved in `HTMLParser` or related files.

Reply:
Removed from xml file and added this programmatically, if RTL is enabled.